index.js

```diff
@@ -204,6 +204,16 @@ function bootstrap(options) {
   app.use(mixins());
   app.use(markdown(config.markdown));
 
+  if (config.getAccessibility === true) {
+    deprecate(
+      '`getAccessibility` option is deprecated and may be removed in future versions.',
+      'Use `pages` to define static cookies page.'
+    );
```

Comment:
this is an interesting one because you are building this into HOF but on the other hand it could be deprecated.  So the ideal is to have a custom pages page?

Reply:
I've added the deprecate method because we do that for terms and cookies. It is just to make it consistent. It may be we keep these and just remove those deprecate messages at some point.